extras/derbynet.conf

```diff
@@ -29,5 +29,32 @@
 # BARCODE_SCANNER_DEV
 
 # For derby-timer.sh script:
-# TIMER_USER=Timer
-# TIMER_PASSWORD=
+# TIMER_ARGS passes arguments to derby-timer.jar. Normal arugments can be used. See usage below.
+# TIMER_ARGS="AddArgumentsHere"
+
+# Usage: [options] <base-url>
+# 
+# -h or -help or --help: This message
+# -v: Show version
+# -x: Run headless, without gui.
+# -logdir <directory>: write log files in <directory> instead of the current directory.
+# -t: Trace non-heartbeat messages sent
+# -th: Trace heartbeat messages sent
+# -r: Show responses to traced messages
+# -u <user>: Specify username for authenticating to web server
+# -p <password>: Specify password for authenticating to web server
+# -n <port name>: Use specified port name instead of searching
+# -min-gate-time <milliseconds>: Ignore gate transitions shorter than <milliseconds>
+# -ignore-place: Discard any place indications from timer
+# -d <device name>: Use specified device instead of trying to identify
+# -delay-reset-after-race <nsec>: how long after race over before timer will be reset, default 10, for SmartLine, DerbyMagic, NewBold, and BertDrake
+# -simulate-timer: Simulate timer device (for testing)
+#    -lanes <n>: Specify number of lanes to report
+#    -pace <nsec>: Staging pace (seconds between heats)
+# -simulate-host: Exercise timer with simulated host
+#    -lanes <n>: Specify number of lanes for scheduling
+# 
+# Experimental flags for The Judge only:
```

Comment:
The command line arguments for the derby-timer.jar do change from time to time, so copying them here risks this comment getting out of date.

I'm thinking you'd be better off talking about just the more important and stable options, and referring the user to the -h command for more up-to-date and detailed information.